lib/routes/react/blog.ts

```diff
@@ -0,0 +1,56 @@
+import type { DataItem, Route } from '@/types';
+import { parseDate } from '@/utils/parse-date';
+import { load } from 'cheerio';
+import ofetch from '@/utils/ofetch';
+
+const handler: Route['handler'] = async () => {
+    const data = await ofetch('https://react.dev/blog');
+
+    const $ = load(data);
+
+    const item = await Promise.all(
+        $('a[href^="/blog/"]')
+            .toArray()
+            .slice(0, 20)
+            .map<Promise<DataItem>>(async (item) => {
+                const $ = load(item);
+
+                const title = $('h2').text();
+                const link = `https://react.dev${item.attribs.href}`;
+                const date = $('div > div:nth-child(2) > div:nth-child(1)').text(); // not reliable, but works for now
+
+                const data = await ofetch(link);
+
+                return {
+                    title,
+                    link,
+                    description: load(data)('article div:nth-child(2)').html() ?? '',
+                    pubDate: parseDate(date),
+                };
```

Comment:
Please [cache](https://docs.rsshub.app/joinus/advanced/use-cache) this request, since the content is a less frequently updated content.